src/uct/ib/base/ib_iface.c

```diff
@@ -205,6 +205,28 @@ ucs_config_field_t uct_ib_iface_config_table[] = {
    "Reverse Service level. 'auto' will set the same value of sl\n",
    ucs_offsetof(uct_ib_iface_config_t, reverse_sl), UCS_CONFIG_TYPE_ULUNITS},
 
+  {"SEND_PRE_OVERHEAD", UCS_VALUE_AUTO_STR,
+   "Time spent to prepare message request and pass it to the hardware or\n"
+   "system software layers", 0,
+   UCS_CONFIG_TYPE_KEY_VALUE(UCS_CONFIG_TYPE_TIME_UNITS,
+       {"common", "common overhead for all operation types",
+        ucs_offsetof(uct_ib_iface_config_t, send_overhead.pre)},
+       {"bcopy", "additional overhead to allocate send descriptor for\n"
+                 "buffered operation",
+        ucs_offsetof(uct_ib_iface_config_t, send_overhead.pre_bcopy)},
+       {NULL},)},
+
+  {"SEND_POST_OVERHEAD", UCS_VALUE_AUTO_STR,
```

Comment:
maybe it should be a common set of parameters for all transports (so defined in `uct_iface_config_table`)

Reply:
I thought about this. But I noticed that previously we took another approach. We replaced common MAX_BCOPY variable by interface specific variable SEG_SIZE.
We can make a common variable. However, it may lead to many unused sub-values in a particular interface.